modules/QnA/resources/qnaService.js

```diff
@@ -243,6 +246,7 @@
             // Setup player ref:
             this.embedPlayer = embedPlayer;
             this.qnaPlugin = qnaPlugin;
+            this.overrideModeratorName =  (qnaPlugin.getConfig('overrideModeratorName')) ? qnaPlugin.getConfig('overrideModeratorName') : false;
```

Comment:
why (qnaPlugin.getConfig('overrideModeratorName')) and not qnaPlugin.getConfig('overrideModeratorName')